pkg/ipamd/ipamd.go

```diff
@@ -329,6 +332,7 @@ func (c *IPAMContext) nodeInit() error {
 	ipamdActionsInprogress.WithLabelValues("nodeInit").Add(float64(1))
 	defer ipamdActionsInprogress.WithLabelValues("nodeInit").Sub(float64(1))
 	var err error
+	ctx := context.Background()
```

Comment:
Make this a `context.TODO()` - This is not an async operation, and ctx should really be passed in from the caller.

Reply:
Ok. Made it a `context.Background()` because it is in a `**Init` function (or) I can pass the context in through the main function.